front/src/routes/integration/all/tp-link/device-page/FoundDevices.jsx

```diff
@@ -0,0 +1,93 @@
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+
+import style from './style.css';
+import { RequestStatus } from '../../../../../utils/consts';
+
+const createDevice = (props, device) => () => {
+  props.createDevice(device);
+};
+
+const GITHUB_BASE_URL = 'https://github.com/GladysAssistant/Gladys/issues/new';
+
+const createGithubUrl = device => {
+  const title = encodeURIComponent(`Philips Hue: Add device ${device.model}`);
```

Comment:
This code is related to Philips Hue integration.

Reply:
Fixed